### PR TITLE
Refector RpmInstallCalicoMesos guide

### PR DESCRIPTION
**Pull request by TrimBiggs**

### CODE REVIEW

File: docs/mesos/RpmInstallCalicoMesos.md [hunk 1]
```diff
@@ -20,25 +20,51 @@ This RPM installation includes and installs:
...
-> Follow our [Mesos Cluster Preparation guide](MesosClusterPreparation.md#Install) if you do not already have an instance of etcd running to walk through installing etcd with Docker.
+Open `/etc/default/mesos-slave` set the`ETCD_AUTHORITY` variable to the correct value.  Your file should now looke like this:
+```
+MASTER=zk://zookeeper:2181
+ETCD_AUTHORITY=<ETCD_IP:ETCD_PORT>
+```
 
-    sudo systemctl start calico-mesos.service
-    sudo systemctl start mesos-slave.service
+> Follow our [Mesos Cluster Preparation guide](MesosClusterPreparation.md#install-zookeeper-and-etcd) if you do not already have an instance of etcd running to walk through installing etcd with Docker.
```
Comment (djosborne) [inline docs/mesos/RpmInstallCalicoMesos.md]: Well it was not you that wrote this, but this sentence is kinda weird. trim off the end:
`Follow our [Mesos Cluster Preparation guide](MesosClusterPreparation.md#install-zookeeper-and-etcd) if you do not already have an instance of etcd running.`

Comment (TrimBiggs) [inline docs/mesos/RpmInstallCalicoMesos.md]: I think I did write that in the last PR ;)

File: docs/mesos/RpmInstallCalicoMesos.md [hunk 2]
```diff
@@ -20,25 +20,46 @@ This RPM installation includes and installs:
...
+Open `/etc/default/mesos-slave` set the`ETCD_AUTHORITY` and `MASTER` 
+variables to the correct values.  Your file should now look like this:
+```
+MASTER=zk://<ZOOKEEPER_IP>:<ZOOKEEPER_PORT>
+ETCD_AUTHORITY=<ETCD_IP>:<ETCD_PORT>
+```
 
-    sudo systemctl start calico-mesos.service
-    sudo systemctl start mesos-slave.service
+> Follow our [Mesos Cluster Preparation guide](MesosClusterPreparation.md#install-zookeeper-and-etcd) if you do not already have an instance of etcd running.
```
Comment (djosborne) [inline docs/mesos/RpmInstallCalicoMesos.md]: `...instances of etcd and zookeeper running`

**Comment by djosborne** [comment]
One thing, then LGTM :shipit:

**Pull request merged**